Specs/DataSources/ModelVisualizerSpec.js

```diff
@@ -292,6 +292,32 @@ defineSuite([
         expect(state).toBe(BoundingSphereState.FAILED);
     });
 
+    it('Fails bounding sphere when model fails to load.', function() {
+        var entityCollection = new EntityCollection();
+        visualizer = new ModelVisualizer(scene, entityCollection);
+
+        var time = JulianDate.now();
+        var testObject = entityCollection.getOrCreateEntity('test');
+        var model = new ModelGraphics();
+        console.log(model);
+        testObject.model = model;
+
+        testObject.position = new ConstantProperty(new Cartesian3(5678, 1234, 1101112));
+        model.uri = new ConstantProperty("/path/to/incorrect/file");
```

Comment:
Style: use single quotes:

`model.uri = new ConstantProperty('/path/to/incorrect/file');`

Reply:
Ah, must have left the console log in there when I was debugging! Sorry!

Will also fix the quotes.